lib/commands/context.js

```diff
@@ -353,6 +368,7 @@ helpers.setupNewChromedriver = async function setupNewChromedriver (opts, curDev
     bundleId: opts.chromeBundleId,
     useSystemExecutable: opts.chromedriverUseSystemExecutable,
     disableBuildCheck: opts.chromedriverDisableBuildCheck,
+    isAutodownloadEnabled: this.isChromedriverAutodownloadEnabled(),
```

Comment:
I actually have a concern now. How should this feature work when `mappingPath` and `executableDir` are set? Should we set `isAutodownloadEnabled` to `false` if they are passed?

Reply:
they can work together: https://github.com/appium/appium-chromedriver/pull/131/files#diff-df03902a1655a73e18bdf5877e4a8de4R274

Reply:
Gotcha. Thanks for explaining